src/index.js

```diff
@@ -19,3 +21,4 @@ export {Tooltip} from "./Tooltip";
 export {Label} from "./Label";
 export {FileInput} from "./FileInput/FileInput";
 export * from "./InfoPanel";
+export {Number};
```

Comment:
ahh gotcha. perhaps worth adding a comment for `Number` since its a special case!